src/coord/src/catalog/builtin.rs

```diff
@@ -793,6 +804,7 @@ lazy_static! {
             Builtin::Table(&MZ_SOURCES),
             Builtin::Table(&MZ_SINKS),
             Builtin::Table(&MZ_VIEWS),
+            Builtin::Table(&MZ_MAP_TYPES),
```

Comment:
Immaterial, but do we expect to a table for each type of type (e.g. map, enum, range)?

Reply:
Yep!